liquibase-standard/src/main/java/liquibase/changelog/RanChangeSet.java

```diff
@@ -116,10 +116,10 @@ public String toString() {
     }
 
     public boolean isSameAs(ChangeSet changeSet) {
-        return this.getId().equalsIgnoreCase(changeSet.getId())
+        String normalizedFilePath = DatabaseChangeLog.normalizePath(this.getChangeLog());
```

Comment:
Hey @Tylorjg ! I just noticed those changes to RanChangeSet (moving the normalizedFilePath from isSamePath to isSameAs methods. Seems they were part of some other tests but are not necessary here anymore - so could we rollback them ? The idea of calling normalize path at method isSamePath is that if author or id doesn't match them we never reach this expensive normalization.